### DISCUSSION TITLE
Task 1: Submission Process and LeaderBoard

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#77] [General]
Can You kindly elaborate the task 1 Submission Process and How to access the LeaderBoard?

**Comment by sarthakpati** [comment]
Hey,

We don't have a leaderboard (still undergoing internal discussion on the best way to structure it). And for the submission, once you start the paper submission, we will reach out with further details on submitting your algorithm. 😄
